### PR TITLE
Tickets/dm 38727

### PR DESCRIPTION
**Pull request by leannep**
Update repo to have latest buid workflows and change test plan

### CODE REVIEW

File: DMTR-231.tex [hunk 1]
```diff
@@ -23,27 +23,27 @@
...
+\date{\date}
+\author{Leanne Guy}
 
 \input{history_and_info.tex}
 
 
 \setDocAbstract{
-This is the test plan and report for
-\textbf{ ComCam OPS Readiness} (LDM-503-11a),
+This is the test plan an report for \textbf{All P1a DM requirements verified}\,(LDM-503-19a),
```
Comment (jeffcarlin) [inline DMTR-231.tex]: an --> and

File: DMTR-231.tex [hunk 2]
```diff
@@ -53,36 +53,25 @@ \section{Introduction}
...
+\citeds{LDM-639}: Data Management Acceptance Test Specification \\
+\citeds{LSE-61}: Data Management Requirements Specification \\
 
 
 \subsection{Document Overview}
 \label{sect:docoverview}
 
 This document was generated from Jira, obtaining the relevant information from the
-\href{https://jira.lsstcorp.org/secure/Tests.jspa\#/testPlan/LVV-P76}{LVV-P76}
+\href{https://jira.lsstcorp.org/secure/Tests.jspa\#/testPlan/LVV-P76}{LVV-P72}
```
Comment (jeffcarlin) [inline DMTR-231.tex]: This seems to link to LVV-P76?

That must be why the last sentence of Sec. 1.4 says "The current status of test plan LVV-P76 in Jira is Completed.", and the Section 2 material is all from LVV-P76.

**Review by jeffcarlin — APPROVED** [review]
Approved once the LVV-P76 --> LVV-P72 change is fixed.

File: DMTR-231.tex (hunk 1 above)
Comment (leannep) [reply DMTR-231.tex]: argh - did not fix the link, only the text

File: DMTR-231.tex (hunk 2 above)
Comment (leannep) [reply DMTR-231.tex]: argh  - wrong cycle too

**Pull request merged**